pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -1106,9 +1109,51 @@ func handleCreate(ctxArg interface{}, key string,
 	log.Functionf("handleCreate(%v) for %s",
 		config.UUIDandVersion, config.DisplayName)
 
+	// Load the UUIDs of the apps that were previously (before the reboot) in the ACTIVE state.
+	activeAppsUUIDs, err := loadActiveAppInstanceUUIDs()
+	if err == nil {
+		// Check if the app is in the active list
+		log.Functionf("config UUID: %s", config.UUIDandVersion.UUID.String())
```

Comment:
```suggestion
		
```
The UUID and Version is already logged here: https://github.com/lf-edge/eve/pull/4707/files#diff-6ac92d68c9dd6b3a3052cf0959aa71795adcabc49ca12faae647c9fe27f6340dR1109

Or do you want to log something like:
```suggestion
		log.Functionf("load active app instances UUIDs was successful")
```

Reply:
Fine for me.